iap/README.md

```diff
@@ -76,6 +76,26 @@ service account private key can impersonate that account!
    ```
 2. Copy `validate_jwt.py` into your application.
 
+## Using generate_self_signed_jwt
+
+### Self-signed JWT with IAM Credentials API
+
+1. Install the libraries listed in `requirements.txt`, e.g. by running:
+   ```
+   virtualenv/bin/pip install -r requirements.txt
+   ```
+2. Call sign_jwt with service account email and target resource url.
+3. Use resulting token to access application.
```

Comment:
Markdown formatting, as well please give a neutral command that is not dependent on a specific setup, such as invoking pip via Python rather than via a specific venv setup.
```suggestion
1. Install the libraries listed in `requirements.txt`, e.g. by running:
   ```
   python3 -m pip install -r requirements.txt
   ```
1. Call sign_jwt with service account email and target resource url.
1. Use resulting token to access application.
```